docs/software-development-kits/java/middleware.mdx

```diff
@@ -0,0 +1,133 @@
+---
+sidebar_label: Spring Middleware
+title: Spring Middleware
+description: Aserto Middleware - Java - Installing and using Spring Middleware
+---
+
+The `AsertoFilter` is a Java library that provides a Spring Security filter that enables the use of Aserto's authorization service in projects that use Spring.
+
+## Requirements
+- Java 8  or newer
+- Spring Boot 2.7.x
+- Spring Security 5.7.x
+
+### Installation
+Add the fallowing dependency to your `pom.xml` file:
+
+```xml
+<dependency>
+    <groupId>com.aserto</groupId>
+    <artifactId>aserto-spring</artifactId>
+    <version>0.0.2</version>
+</dependency>
+```
+
+The latest version is available on [Maven Central](https://search.maven.org/artifact/com.aserto/aserto-spring).
+
+And them execute:
+```bash
+mvn clean package
+```
+
+In order for Spring to discovery the filter, we need to add the `@ComponentScan` annotation to the main class of the application.
+```java
+@ComponentScan("com.aserto")
+```
+
+## Configuration
+
+The authorization filter accepts the fallowing configuration parameters:
+
+| Parameter name | Default Value | Description |
```

Comment:
I would separate the config elements that are only relevant in the hosted environment. Perhaps split the table in two and add:

```
The following configuration settings are available when connecting the middleware to Aserto's hosted authorizer service:
| Parameter name | Default Value | Description |
aserto.authorizer.apiKey | N/A | API key to use authenticate with the hosted authorizer service |
| aserto.tenantId | N/A | The tenant ID to use when connecting to the hosted authorizer service |
| aserto.authorizer.policyName | N/A | The name of the policy in which decisions are evaluated |
| aserto.authorizer.policyLabel | N/A | The instance label of the policy in which decisions are evaluated |
```

Reply:
Splitted the config as suggested